Ex2/ex2.js

```diff
@@ -0,0 +1,26 @@
+//Scrieți o funcție pentru a genera un număr aleatoriu mai mare sau egal cu o variabilă start și strict mai mic decât o variabilă end . 
+//Aceste două variabile sunt capetele intervalului pentru numărul general: [start, end)
+//Funcția getRandom(start, end) returnează un număr întreg aleator generat în intervalul definit de variabilele de început și sfârșit furnizate.
+
+
+function getRandom(start, end) {
+    if (start >= end) {
+        return undefined;
```

Comment:
se poate chiar de aruncat o eroare:
`throw "start cannot be bigger than end"`

Reply:
vad ca mai jos ai acea eroare: deci o poti returna direct aici:

throw "Nu s-a putut genera un număr aleatoriu din cauza valorilor invalide pentru start și end."